### PR TITLE
Bugfix/past date selection

### PR DESCRIPTION
**Pull request by burjulius**

### CODE REVIEW

File: src/app/components/availability/availability.directive.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ class AvailabilityController {
     this.changePasswordForm.newPasswordConfirm = "";
     this.showChangePassword = false;
     this.datepickerOptions = {
+      "minDate": new Date().setHours(0,0,0),
```
Comment (karolisgrinkevicius) [inline src/app/components/availability/availability.directive.js]: As far as I know there is moment.js library for date and time.

Comment (burjulius) [inline src/app/components/availability/availability.directive.js]: Yeah, I know that. I tried to use `"minDate": this.moment().format("YYYY-MM-DD"),`, but it does not work for me...

Comment (karolisgrinkevicius) [inline src/app/components/availability/availability.directive.js]: Try `<reference.to.momentjs()>.startOf('day')`

Comment (karolisgrinkevicius) [inline src/app/components/availability/availability.directive.js]: Basically if you need to get full timestamp use `this.moment().startOf("day").valueOf();`

Comment (burjulius) [inline src/app/components/availability/availability.directive.js]: @karolisgrinkevicius Thanks! Now works like a charm 👍

**Pull request merged**